portfolio/src/main/webapp/dogs.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>My Dogs</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <header>
+      <div class="dogheader">
+        <h1>My Dogs</h1>
+      </div>
+    </header>
+    <nav>
+      <div class="navbar">
+        <a href="index.html">Home</a>
+        <a href="dogs.html">Dogs</a>
+        <a href="sports.html">Sports</a>
+        <a href="baking.html">Baking</a>
+      </div>
+    </nav>
+    <div class="dogrow">
+      <div class="dogcolumn"  style="background-color: #cccccc;">
+        <h1>Pepper</h1>
+          <p>
+             On Christmas day in 2010, my parents surprised me and my sister with Pepper! She's a Yorkie-Poo, which is a mix between a yorkshire terrier and a poodle.
+             She was named Pepper because when she was smaller, her coat was the color of pepper, so we thought it was fitting.
+             Pepper is now 9 years old, her birthday is on October 24th where she'll hit double digits this year.
+             She's about the size of a toddler and knows tricks like sit, down, paw, roll over, stay, walk, and more.
+             One of her favorite things to do is sit on the couch and look out the window, barking at anything that passes.
+             Pepper likes to play fetch and run around at the park, but sometimes she isn't too friendly with other dogs.
+             She's an amazing dog and I love her so much.
+          </p>
+          <p><b>Click here to see a random picture of Pepper</b></p>
+          <button onclick="randomPepperImage()">Pepper</button>
+          <div id="random-image-container"></div>
+      </div>
+      <div class="dogcolumn" style="background-color: pink;">
+        <h1>Ginger</h1>
+          <p>
+            In October 2019, my sister rescued a dog and named her Ginger. She's 2 years old, but her birthday is unknown so instead we'll celebrate her rescue day.
```

Comment:
Is Ginger getting along with Pepper?

Reply:
At first they were both very protective but now they're getting along pretty well. Soon they'll be best friends I hope :)